src/config/configmap.ts

```diff
@@ -114,3 +114,52 @@ export const JS_TO_RUBY_MAPPING: Record<keyof AppsignalOptions, string> = {
   statsdPort: "statsd_port",
   workingDirectoryPath: "working_directory_path"
 }
+
+export const BOOL_KEYS: string[] = [
```

Comment:
Nit: with a bit of TypeScript magic, we can ensure the values here are existing keys of `ENV_TO_KEY_MAPPING`, preventing typos.

First, we change `ENV_TO_KEY_MAPPING` from being declared as a specific type:

```ts
export const ENV_TO_KEY_MAPPING: Record<string, keyof AppsignalOptions> = {...}
```

To being constrained to that type, but its actual type being inferred by TypeScript:

```ts
export const ENV_TO_KEY_MAPPING = {...} satisfies Record<string, keyof AppsignalOptions>
```

This means that its type is inferred as having the specific keys in the `ENV_TO_KEY_MAPPING` literal.

And then, we can declare `BOOL_KEYS` (and `STRING_KEYS` and so on) as follows:

```ts
export const BOOL_KEYS: (keyof typeof ENV_TO_KEY_MAPPING)[] = [...]

Reply:
Woudn't doing this negate the need of having a separate test?

I'm not against doing this, but let's have a separate pull request for it. I'd like to push this patch out. :)

Reply:
Go ahead! As I said, none of this is a blocker.

I don't think this negates the need for the test because this only checks that the keys are in the list, not that each key is only in one list.